index.php

```diff
@@ -8,8 +8,31 @@
 $blogModel = new BlogModel($db);
 $blogs = $blogModel->getAllPosts();
 
-?>
 
+$selected = '';
+if(isset($_POST['Filter'])) {
+    $selected = $_POST['sort'];
+    if ($selected == 'oldest') {
+        $blogs = array_reverse($blogs);
+    }
+    if ($selected == 'liked') {
+        usort($blogs, function($a , $b)
+        {
+            return $b->likes - $a->likes;
+        });
+    }
+
+    if ($selected == 'disliked') {
+        usort($blogs, function($a , $b)
+        {
+            return $b->dislikes - $a->dislikes;
+        });
```

Comment:
Same thing, relaly it would be better to get the DB to sort for you, but I'm happy enough with this for now.